extensions/wikia/Discussions/EnableDiscussionsController.class.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+use Email\Fatal;
+
+class EnableDiscussionsController extends \WikiaController {
+
+	const ENABLE_DISCUSSIONS_NAVIGATION = 'wgEnableDiscussionNavigation';
+	const ENABLE_DISCUSSIONS = 'wgEnableDiscussion';
+	const ENABLE_FORUMS = 'wgEnableForumExt';
+
+	const P_SITE_ID = 'siteId';
+	const REASON = 'Enabling discussions!';
+
+	public function index() {
+		$this->assertCanAccessController();
+
+		$siteId = $this->request->getParams()[self::P_SITE_ID];
```

Comment:
PHP will throw a fit if 'siteId' isn't set.  While required, it could be handled better.  You can use `getVar` or in this case, the varient `getInt` to safely retrieve a single value if its there:
```php
$this->request->getInt( self::P_SITE_ID );
```
This would also save you from the `!is_numeric` check since `getInt` will only return an int.